spec/requests/shows_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+load "#{Rails.root}/db/seeds.rb"
+
+RSpec.describe ShowsController, type: :feature do
+  describe "GET shows#index" do
+    it 'renders index and has the header' do
+      visit '/'
+      expect(page).to have_text 'Smartflix'
+    end
+
+    context "when data is seeded" do
+      it 'includes the shows titles' do
+        visit "/"
+          expect(page).to have_text('My Little Pony: A New Generation')
```

Comment:
ok am I blind or.....how is this showing? `app/views/shows/index.html.haml` this seems to be empty :O

Reply:
oh if you ran the specs I guess it would fail because you actually did not implement it yet 😅  sorry about that.